jjb/ci-management/packer.yaml

```diff
@@ -36,13 +36,10 @@
     build-timeout: 90
 
     platforms:
-      - centos-7
```

Comment:
Are these centos images no longer supported?

Reply:
That's correct. CentOS 7 is very out of date, and it's become more difficult to support newer forks (Stream, Rocky) for 8 and 9, so we're pretty much leaving it behind. Many projects are also moving to Github Actions, which defaults to Ubuntu, so we're primarily focused on supporting Ubuntu moving forward.